tests/jest-extensions-babel-transform.js

```diff
@@ -5,7 +5,7 @@ const babelJest = require( 'babel-jest' );
 
 module.exports = babelJest.createTransformer( {
 	presets: [
-		require.resolve( '@automattic/calypso-build/babel/default' ),
+		[ require.resolve( '@automattic/calypso-build/babel/default' ), { modules: 'commonjs' } ],
```

Comment:
I believe the calypso-build jest preset is being overwritten. Rather than making any changes here (maybe this entire file can be removed?), let's try removing the transforms here:

https://github.com/Automattic/jetpack/blob/555b14071a8edf95a506ff51bb855f1a03a1232a/jest.config.js#L9-L14

Reply:
Like so?

```diff
diff --git a/jest.config.js b/jest.config.js
index 10058a9d6..b17ab648b 100644
--- a/jest.config.js
+++ b/jest.config.js
@@ -6,10 +6,4 @@ const path = require( 'path' );
 module.exports = {
        preset: '@automattic/calypso-build',
        roots: [ '<rootDir>/extensions/' ],
-       transform: {
-               '\\.[jt]sx?$': path.join( __dirname, 'tests', 'jest-extensions-babel-transform' ),
-               '\\.(gif|jpg|jpeg|png|svg|scss|sass|css)$': require.resolve(
-                       '@automattic/calypso-build/jest/transform/asset'
-               ),
-       },
 };
```

That gives me two `ReferenceError: React is not defined` errors when running `npm run test-extensions` (i.e. `jest`), in `extensions/blocks/markdown/test/markdown-renderer.js` and `extensions/blocks/tiled-gallery/layout/mosaic/test/index.js` :confused:

Reply:
It's because we're missing the [`wordpress-element` transform from `jest-extensions-babel-transform`](https://github.com/Automattic/jetpack/blob/555b14071a8edf95a506ff51bb855f1a03a1232a/tests/jest-extensions-babel-transform.js#L9).

Did I say that I hate `@wordpress/element`? :angry:

Reply:
After a call with @sirreal, we decided to leave things as they are, since there's no straightforward way to simplify this.